python/tvm/relay/frontend/sklearn.py

```diff
@@ -120,7 +120,19 @@ def _InverseLabelTransformer(op, inexpr, dshape, dtype, columns=None):
     """
     Identity transformation of the label data. The conversion to string happens in runtime.
     """
-    return _op.copy(inexpr)
+    if len(dshape) > 2:
```

Comment:
Will dshape come from the DataConfig input?

Reply:
Yes. I think that's the only way we can distinguish binary/multiclass classification. Do you have better ideas?